### PR TITLE
SRVKS-573: Updating autoscaling docs

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVKS-573 (related groundwork; there will be more PRs to add the ODC content and close this issue)

Applies for OCP 4.6+

Direct preview link: https://deploy-preview-38284--osdocs.netlify.app/openshift-enterprise/latest/serverless/autoscaling/serverless-autoscaling.html

### CODE REVIEW

File: serverless/autoscaling/serverless-autoscaling-scale-bounds.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-autoscaling-scale-bounds
+
+toc::[]
+
+You can set scale bounds for an application to help prevent cold starts or control computing costs. Scale bounds determine the minimum and maximum numbers of pods that can serve an application at any given time. The `minScale` and `maxScale` annotations can be used to configure scale bounds.
+
+[NOTE]
+====
+The following annotations apply for the full lifetime of a revision. Even when a revision is not referenced by any route, the minimal pod number specified by the `minScale` annotation is still provided. Non-routeable revisions may be garbage collected, which enables Knative to reclaim the resources.
```
Comment (markusthoemmes) [inline serverless/autoscaling/serverless-autoscaling-scale-bounds.adoc:13]: AFAIK this is no longer true. MinScale no longer applies to non-routable revisions I think.

**Review by markusthoemmes — COMMENTED** [review]
LG modulo one comment

File: serverless/autoscaling/serverless-autoscaling-scale-bounds.adoc (hunk 1 above)
Comment (abrennan89) [reply serverless/autoscaling/serverless-autoscaling-scale-bounds.adoc:13]: Thank you, I really wanted to remove this but wasn't sure if it was still relevant 😄

**Comment by jrangelramos** [comment]
LGTM
QE Approved

File: modules/serverless-autoscaling-minscale.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
+[id="serverless-autoscaling-minscale_{context}"]
+= Configuring the minScale annotation
+
+If the `minScale` annotation is not set, pods scale to zero, or to 1 if `enable-scale-to-zero` is set to `false` in the `config-autoscaler` config map.
+
+.Example service spec with `minScale` configured
```
Comment (bmcelvee) [inline modules/serverless-autoscaling-minscale.adoc:6]: For consistency, would you mind formatting _Configuring the maxScale annotation_ and _Configuring the maxScale annotation_ as procedures?

**Comment by bmcelvee** [comment]
Just a couple of questions and suggestions. Otherwise, LGTM! 

Also, apologies for starting a review. I just meant to leave comments. :woman_facepalming:

**Comment by abrennan89** [comment]
Thanks for the comments @bmcelvee - decided to rework some of the docs because I think I can improve them a bit.
I'll send them for another peer review once updated. Thanks!

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**